test/unit/util/sfdcUrlTest.ts

```diff
@@ -108,7 +108,7 @@ describe('util/sfdcUrl', () => {
       emitWarningSpy.restore();
     });
 
-    it('listens for the insecure http signal', () => {
+    it('emits the insecure http signal', () => {
```

Comment:
@maggiben should a test be added to make sure emit is only called once?

Reply:
yes...maybe instantiate the SfdcUrl again and verify the count is still 1.

Reply:
Good idea, test created for single call-count